src/components/m-config-vue3.vue

```diff
@@ -20,37 +20,50 @@ export default defineComponent({
   components: {
     Vue3JsonEditor, 
   },
-  setup () {
+  props: ['session'], 
```

Comment:
its probably better if we leave session of of this, and just pass session.params objects (or some other json), the full session has information that's not needed here.